library/core/src/num/f128.rs

```diff
@@ -908,7 +908,7 @@ impl f128 {
     /// ```
     #[inline]
     #[unstable(feature = "f128", issue = "116909")]
-    #[rustc_const_unstable(feature = "const_float_bits_conv", issue = "72447")]
+    #[rustc_const_unstable(feature = "f128_const", issue = "116909")]
```

Comment:
I tried that first and somehow it doesn't work. Probably something the const stability checks are doing doesn't work quite right when using a language feature (rather than a library feature) as the feature gate.

Reply:
Here's an issue tracking this problem with const stability: https://github.com/rust-lang/rust/issues/129656